### ISSUE TITLE
Request: clear entire history

### ISSUE THREAD

**Issue by zpuskas** [EvanRespaut/Equate#37]
It would be nice to be able to clear the full history with a single button or from settings. After doing a lot of conversions deleting them one-by-one is too time consuming.

**Comment by EvanRespaut** [comment]
If you hold down the clear key, a dialog appears either the option to clear the history.

**Comment by EvanRespaut** [comment]
Yeah that's a good idea, maybe I can find or create a symbol that represents a reset...

**Comment by ChiefMikeK** [comment]
🆑 U+1F191 SQUARED CL alias=clear
